identity/app/views/fragments/form/switchboardLabel.scala.html

```diff
@@ -0,0 +1,63 @@
+@import _root_.form.IdFormHelpers.{Checkbox}
+
+@(
+    title: String,
```

Comment:
Should `title` be named `consentText`?

Reply:
the checkbox component will also be reused for newsletters so it should use its own naming imo. A pattern react uses a lot if creating a middle component that just wraps the original one and maps the model api to the component api, right now @v2consentCheckbox is doing that job